injectionapi/socketlabsclient.go

```diff
@@ -47,6 +55,7 @@ func CreateClient(serverID int, apiKey string) ISocketlabsClient {
 		ServerID:    serverID,
 		APIKey:      apiKey,
 		EndpointURL: endpointURL,
+		RequestTimeout: 120,
```

Comment:
Set the default value as a const like endpointURL on line 16

Reply:
added const